go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/asticode/go-astilectron
```

Comment:
I'd rather not complexify the PR with adding module support. Can you remove this and make it in another PR?

Reply:
Removed additions of go.mod, .gitignore and Makefile - will leave for future PR.